pkcli.roles AssertionError: invalid simulation type=flash

This happened on beta. We don't have flash configured yet so it's not a valid sim so `pkcli.roles` fails when trying to add a user role. Trying to do #2408.

```
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/simulation_db.py", line 170, in examples
    sirepo.sim_data.get_class(app).resource_path(_EXAMPLE_DIR),
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/sim_data/__init__.py", line 63, in get_class
    ) else type_or_data
  File "/home/vagrant/.pyenv/versions/3.7.2/envs/py3/lib/python3.7/site-packages/sirepo/template/__init__.py", line 49, in assert_sim_type
    assert is_sim_type(sim_type), 'invalid simulation type={}'.format(sim_type)
AssertionError: invalid simulation type=flash
```

--- a/sirepo/pkcli/admin.py
+++ b/sirepo/pkcli/admin.py
@@ -65,8 +65,8 @@ def audit_proprietary_lib_files(*uid):
                 pass
 
     server.init()
-    t = feature_config.cfg().proprietary_sim_types
-    if not t or not t.issubset(feature_config.cfg().sim_types):
+    t = feature_config.cfg().proprietary_sim_types.intersection(feature_config.cfg().sim_types)
+    if not t:
         return
     for u in uid or auth_db.all_uids():
         _audit_user(u, t)

--- a/sirepo/pkcli/admin.py
+++ b/sirepo/pkcli/admin.py
@@ -66,7 +66,7 @@ def audit_proprietary_lib_files(*uid):
 
     server.init()
     t = feature_config.cfg().proprietary_sim_types
-    if not t:
+    if not t or not t.issubset(feature_config.cfg().sim_types):
         return
     for u in uid or auth_db.all_uids():
         _audit_user(u, t)

